packages/rum-core/test/common/apm-server.spec.js

```diff
@@ -398,4 +401,28 @@ describe('ApmServer', function() {
     ]
     expect(result).toEqual(expected)
   })
+
+  if (isVersionInRange(testConfig.stackVersion, '7.3.0')) {
+    it('should fetch remote config', async () => {
+      var config = await apmServer.fetchConfig('nonexistent-service')
+      expect(config).toEqual({})
+      try {
+      } catch (e) {
+        debugger
+        console.log(e)
+      }
+
+      config = await apmServer.fetchConfig(
+        'nonexistent-service',
+        'nonexistent-env'
+      )
+      expect(config).toEqual({})
+
+      try {
+        config = await apmServer.fetchConfig()
+      } catch (e) {
+        expect(e).toBe('serviceName is required for fetching central config.')
```

Comment:
should be removed here based on the validation logic.

Reply:
Responded above.